demo/src/app/components/+dropdown/dropdown-section.list.ts

```diff
@@ -72,17 +73,26 @@ export const demoComponentContent: ContentSection[] = [
         outlet: DemoDropdownSplitComponent
       },
       {
-        title: 'Manual triggers',
+        title: 'Manual triggering',
         anchor: 'triggers-manual',
         component: require('!!raw-loader?lang=typescript!./demos/triggers-manual/triggers-manual.ts'),
         html: require('!!raw-loader?lang=markup!./demos/triggers-manual/triggers-manual.html'),
-        description: `<p>Dropdown can be triggered in two ways:
-          <ul>
-            <li>by toggling <code>isOpen</code> property</li>
-            <li>by <code>show</code>/<code>hide</code> methods from directive</li>
-          </ul></p>`,
+        description: `<p>Dropdown can be triggered by <code>show</code>, <code>hide</code> and
+          <code>toggle</code> methods from directive
+          <br>
+          Use method <code>toggle(true)</code> if you want create toggler, or <code>toggle(false)</code>
+          if you want only close opened dropdown.
```

Comment:
if you want only close opened dropdown -> if you only want to close the dropdown